src/Symfony/Component/Console/Attribute/Argument.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Attribute;
+
+use Symfony\Component\Console\Completion\CompletionInput;
+use Symfony\Component\Console\Completion\Suggestion;
+use Symfony\Component\Console\Exception\LogicException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+
+#[\Attribute(\Attribute::TARGET_PARAMETER)]
+class Argument
+{
+    private const ALLOWED_TYPES = ['string', 'bool', 'int', 'float', 'array'];
+
+    private ?int $mode = null;
+
+    /**
+     * Represents a console command <argument> definition.
+     *
+     * If unset, the `name` and `default` values will be inferred from the parameter definition.
+     *
+     * @param string|bool|int|float|array|null                               $default         The default value (for InputArgument::OPTIONAL mode only)
+     * @param array|callable-string(CompletionInput):list<string|Suggestion> $suggestedValues The values used for input completion
```

Comment:
I suggest using `array<string|Suggestion>` to document the type of accepted values for the array.